### ISSUE TITLE
http.destroy fails

### ISSUE THREAD

**Issue by kevcrumb** [knaccc/i2p-zero#19]
A couple of days ago, I ran `launch.sh`, but `all.list` wouldn't show the tunnels I had previously defined. I executed the `create` commands and ended up with multiple tunnels for the same port, none of which can now be deleted.

```
# ./tunnel-control.sh all.list | jq '.tunnels[]'
{
  "port": "4444",
  "state": "open",
  "type": "http"
}
{
  "port": "4444",
  "state": "open",
  "type": "http"
}
...
```

```
# ./tunnel-control.sh http.destroy 4444
# ./tunnel-control.sh client.destroy 4444
OK
```

... but **all.list** still shows all. Nothing was actually deleted.

**Comment by knaccc** [comment]
Are there error messages in wrapper.log that appear at the same time as you issue the destroy commands?

**Comment by knaccc** [comment]
I think this should be fixed now in v1.10. Please re-open the ticket if you still have problems.
